src/components/FormModal/FormModal.js

```diff
@@ -39,7 +39,7 @@ const FormModal = ({ user, toggleFormModal }) => {
 
   return (
 
-    <article className="form-modal">
+    // <article className="form-modal">
```

Comment:
Good catch - looks like this wrapping article isn't really necessary. Removing that now to push up.